### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by geramirez**
Updating documentation with haystack and elasticsearch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ This is a Django app that uses [Postgres](http://www.postgresql.org/), and depen
 
 **Installing Postgres**: You can `brew install postgres` (OS X) or `apt-get install postgresql` (Ubuntu).
 
+**Installing Postgres**: You can `brew install elasticsearch` (OS X) or `apt-get install elasticsearch` (Ubuntu).
```
Comment (khandelwal) [inline README.md]: I think you mean 'Installing Elasticsearch'

File: README.md [hunk 2]
```diff
@@ -106,21 +108,35 @@ If you you get a `could not connect to server` error, you could be experiencing
 psql -d foia -c "CREATE USER foia WITH PASSWORD '<<PASSWORD>>';"
 ```
 
+* Change database password in local_settings.py:
```
Comment (khandelwal) [inline README.md]: We'd removed this earlier. Why's it back?

File: README.md [hunk 3]
```diff
@@ -106,21 +108,35 @@ If you you get a `could not connect to server` error, you could be experiencing
...
 ```
 
-* Finally, launch the server locally:
+### Elasticsearch setup
 
+* Start the Elasticsearch server
+```bash
+elasticsearch
 ```
-django-admin.py runserver
```
Comment (khandelwal) [inline README.md:118]: We're removing this?

File: foia_hub/settings/local_settings.py.example [hunk 4]
```diff
@@ -15,5 +15,14 @@ DATABASES = {
     }
 }
 
+# These are the default setting for haystack.
```
Comment (khandelwal) [inline foia_hub/settings/local_settings.py.example]: s/setting/settings

File: README.md (hunk 3 above)
Comment (geramirez) [inline README.md:118]: Yeah, I think it's in the wrong place under setting up postgres. Maybe I'll move it to a different section.

File: README.md (hunk 2 above)
Comment (geramirez) [inline README.md]: It seemed like a more important part of setup than running the server. I can take it out. Let me know.

File: README.md (hunk 3 above)
Comment (khandelwal) [inline README.md:118]: Ah. Make those changes in 18f/foia-hub then as they are not strictly related to krang.

**Comment by khandelwal** [comment]
So, what's going on with this? You made some non-krang related changes here. Can you move those out to a pull request on 18f/foia-hub instead? This fork is really meant for search related changes only.

**Comment by geramirez** [comment]
@khandelwal ok, I moved out the non-krang updates. I'll make the other changes later on 18F/foia-hub

**Pull request merged**